sale_report_country_state/reports/report_all_channels_sales.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ReportAllChannelsSales(models.Model):
+    _inherit = "report.all.channels.sales"
+
+    state_id = fields.Many2one(
+        comodel_name="res.country.state", string="Partner's State", readonly=True,
```

Comment:
Change to *Customer State* so it's grouped altogether with the other partner's properties

![image](https://user-images.githubusercontent.com/5040182/119458886-7cd63580-bd3d-11eb-8d58-00ec85e61b11.png)